### PR TITLE
THU/scenario_manager_v2_prod-8147

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

File: src/charts/ScenarioManagerTreeList/ScenarioManagerTreeList.js [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
 import { ScenarioUtils } from '@cosmotech/core';
 import useStyles from './style';
 import { ScenarioNode } from '../../cards';
+import { SHRUNK_NODE_HEIGHT, EXPANDED_NODE_HEIGHT } from '../../cards/ScenarioNode/constants';
+
+const initNodesDict = (scenarios) => {
+  const nodesDict = {};
+  scenarios.forEach((scenario) => {
+    nodesDict[scenario.id] = false;
+  });
```
Comment (nborde-CSM) [inline src/charts/ScenarioManagerTreeList/ScenarioManagerTreeList.js:20]: In this case, shouln't we use `map` insteed of `forEach` ?
`const nodesDict = scenario.map((scenario) => {
    const dict = 
        return {[senario.id]: false};
}`

File: src/cards/ScenarioNode/ScenarioNode.js [hunk 2]
```diff
@@ -32,63 +51,172 @@ export const ScenarioNode = ({
...
+    }
+    return classes.statusUnknown;
+  };
+
+  const getStatusIcon = (showLabel) => {
+    const statusClassName = getStatusClassName(classes, scenario.state);
+    const iconClassName = getStatusIconClassName(classes, scenario.state);
+    const status = getTranslatedStatus(labels, scenario.state);
+    let icon = null;
+    if (scenario.state === 'Successful') {
```
Comment (nborde-CSM) [inline src/cards/ScenarioNode/ScenarioNode.js:106]: Switch case insteed of `else if `series ?

**Review by nborde-CSM — APPROVED** [review]
Few little remarks but overall look good to me.

File: src/charts/ScenarioManagerTreeList/ScenarioManagerTreeList.js [hunk 3]
```diff
@@ -24,32 +35,56 @@ export const ScenarioManagerTreeList = (props) => {
     buildScenarioNameToDelete,
     showDeleteIcon,
   } = props;
+  if (buildSearchInfo) {
+    console.log('"buildSearchInfo" prop is deprecated in ScenarioManagerTreeList. Please consider removing this prop.');
```
[the remark this answers is not in the document]
Comment (csm-thu) [reply src/charts/ScenarioManagerTreeList/ScenarioManagerTreeList.js:39]: I started with a `console.warn` but it displayed a huge call stack for every warning in the console browser so I replaced it by a `console.log`. But I guess you're right, a `console.warn` is the right logging type here, so I'll switch it back :)

File: src/charts/ScenarioManagerTreeList/ScenarioManagerTreeList.js (hunk 1 above)
Comment (csm-thu) [reply src/charts/ScenarioManagerTreeList/ScenarioManagerTreeList.js:20]: `scenario.map(...)` returns an array, but we need a dict

**Pull request merged**